vaadin-checkbox-flow-integration-tests/src/main/java/com/vaadin/flow/component/checkbox/tests/CheckboxGroupDemoPage.java

```diff
@@ -54,6 +60,27 @@ protected void initView() {
         addDisabledItems();
         addReadOnlyGroup();
         addComponentWithThemeVariant();
+        addDataViewDemo();
+    }
+
+    private void addDataViewDemo() {
+        CheckboxGroup<String> checkboxGroup = new CheckboxGroup<>();
```

Comment:
Here also demo headers and footer are/is missing.

Reply:
Done.